### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by Grohot9**

### CODE REVIEW

File: homework_2/generate_password.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from flask import Flask
+import random as rnd
```
Comment (mikolaz27) [inline homework_2/generate_password.py]: remove as rnd

File: homework_2/generate_password.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def generate_password() -> str:
+    """
+    from 10 to 20 chars
+    upper and lower case
```
Comment (mikolaz27) [inline homework_2/generate_password.py]: use one file for both functions (views)

File: homework_2/generate_password.py (hunk 1 above)
Comment (Grohot9) [reply homework_2/generate_password.py]: ✅

File: homework_2/generate_password.py (hunk 2 above)
Comment (Grohot9) [reply homework_2/generate_password.py]: ✅

File: homework_2/flask_tasks.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+@app.route("/generate_password")
+def generate_password() -> str:
+    """
+    from 10 to 20 chars
+    upper and lower case
+    """
+    password_length = random.randint(10, 20)
+    available_characters = (string.ascii_lowercase, string.ascii_uppercase, string.digits, string.punctuation)
+    password = "".join([random.choice(available) for available in available_characters])
+    for _ in range(password_length - len(available_characters)):
```
Comment (mikolaz27) [inline homework_2/flask_tasks.py]: you can use random.choices its doing exactly what you need

**Review by mikolaz27 — APPROVED** [review]

**Pull request merged**